test/Macros/SwiftifyImport/CxxSpan/LifetimeboundSpan.swift

```diff
@@ -0,0 +1,28 @@
+
+// REQUIRES: swift_swift_parser
+// REQUIRES: swift_feature_Span
+
+// RUN: %target-swift-frontend %s -enable-experimental-cxx-interop -I %S/Inputs -Xcc -std=c++20 -swift-version 5 -module-name main -disable-availability-checking -typecheck -enable-experimental-feature Span -enable-experimental-feature LifetimeDependence -plugin-path %swift-plugin-dir -dump-macro-expansions 2>&1 | %FileCheck --match-full-lines %s
+
+import CxxStdlib
+import StdSpan
+
+public struct VecOfInt {}
+
+@_SwiftifyImport(.lifetimeDependence(from: 1, to: .return, type: .copy), typeMappings: ["SpanOfInt" : "std.span<CInt>"])
+func myFunc(_ span: SpanOfInt) -> SpanOfInt {
+}
+
+@_SwiftifyImport(.lifetimeDependence(from: 1, to: .return, type: .borrow), typeMappings: ["SpanOfInt" : "std.span<CInt>"])
+func myFunc2(_ vec: borrowing VecOfInt) -> SpanOfInt {
```

Comment:
Does/will ClangImporter import parameters as `borrowing`? For which cases? If not, we could always add `borrowing` in the signature of the generated wrapper.

Reply:
Yes, the plan is to make `ClangImporter` import C++ references as borrowing and (`@addressable`) in all cases to avoid some copies and to be safer in cases where the function returns a dependent pointer/reference.

Reply:
Actually, we have not fully decided yet on all the details whether all or only lifetimebound annotated references would be imported as borrowing. But I think we want to do this transformation in the ClangImporter for sure because the original declaration should also have the right calling convention.

Reply:
Cool 👍